chore(docs): update github action example in managing environment guide (#15629)
supabase/supabase: apps/docs/pages/guides/getting-started/local-development.mdx

Local logs rely on the Supabase Analytics Server. This can be enabled via the CLI configuration, and requires a Google Cloud project and BigQuery access.

<Admonition type="note">

>   The Google Cloud project must have billing enabled. Its dataset location must also be in the US.

Reviewer: ```suggestion
  The Google Cloud project must have billing enabled. The dataset created and managed by Analytics will be within the US.
```
